src/corehost/cli/runtime_config.h

```diff
@@ -26,8 +33,8 @@ class runtime_config_t
     const std::list<pal::string_t>& get_probe_paths() const;
     bool get_patch_roll_fwd() const;
     bool get_prerelease_roll_fwd() const;
-    int get_roll_fwd_on_no_candidate_fx() const;
-    void set_roll_fwd_on_no_candidate_fx(int value);
+    enum roll_fwd_on_no_candidate_fx_option get_roll_fwd_on_no_candidate_fx() const;
```

Comment:
It's unnecessary to prefix all of the uses of the enum with `enum `.

Reply:
Done. Thanks